### DISCUSSION TITLE
Code Review IM-13

### DISCUSSION THREAD

**Discussion by crazysparrow69** [HowProgrammingWorks/LiveQA#53] [Code review]
Розробляю сайт з повноцінним функціоналом https://github.com/crazysparrow69/Course_work
Деплоймент можна глянути тут https://github.com/crazysparrow69/Course-work-deployment
Поки що готова лише головна сторінка з пошукачем, продуманий дизайн всього сайту, анімації та деякі файли "бази", але в майбутньому буде створено багато нового:

- форма реєстрації з можливістю входу/виходу з аккаунта
- можливість зберігати фільми у "обраному" на кожному з аккаунтів окремо
- фільтри для пошуку
- сторінки кожного з фільма/серіалу окремо
- "карусель" фільмів
та багато чого іншого що стукне в голову.
Працювати будемо в команді з трьох чоловік.
Буду вдячний за огляд та коментар щодо ідеї.

**Comment by tshemsedinov** [comment]
https://github.com/crazysparrow69/Course_work/issues/7

**Comment by crazysparrow69** [comment]
Доброго дня, хотів би отримати ревью на оновлений код https://github.com/crazysparrow69/Course_work
З минулого разу доробив: 
- фільтрацію по жанрам на головній сторінці (genres.js);
- модальне вікно (modal_window.js);
- форми реєстрації та входу в аккаунт (forms.js);
- "базу даних" зареєстрованих юзерів (users.js);
-  почав реалізовувати ідею щодо можливості додавати фільми у обране (films_actions.js) та профілю (profile.js);
Буду безмежно вдячний за огляд та поради.
